Resources/Prototypes/_NF/Entities/Structures/Specific/aberrant_flesh.yml

```diff
@@ -71,4 +71,19 @@
         spawn:
           FoodMeat:
             min: 1
-            max: 2
\ No newline at end of file
+            max: 2
+
+- type: entity
+  parent: AberrantFleshDigestiveSack
+  id: AberrantFleshDigestiveSackTest
+  name: assimilation sack
+  suffix: DO NOT MAP
+  components:
+  - type: Timer
+  - type: TimedSpawner
+    prototypes:
+    - SpawnMobAberrantFleshMinerExpeditions
+    chance: 1.0
+    intervalSeconds: 4
+    minimumEntitiesSpawned: 1
+    maximumEntitiesSpawned: 1
```

Comment:
Is this intended to be left as a test entity?  Cleaned up?

Reply:
Yeah, I think this might be useful for test ting later down the line.